src/helpers.js

```diff
@@ -109,6 +109,10 @@ export function rotated(point, center, angle) {
   };
 }
 
+function isImage(content) {
```

Comment:
Wouldn't `isImageOrCanvas` be a more appropiate name?

Reply:
yes, let me update it.

Reply:
done!